packages/xdl/src/Project.ts

```diff
@@ -1220,14 +1385,14 @@ async function uploadAssetsAsync(projectRoot, assets) {
 async function getConfigAsync(
   projectRoot: string,
   options: {
-    current?: boolean,
-    mode?: string,
-    platform?: string,
-    expIds?: Array<string>,
-    type?: string,
-    releaseChannel?: string,
-    bundleIdentifier?: string,
-    publicUrl?: string,
+    current?: boolean;
+    mode?: string;
+    platform?: 'android' | 'ios' | 'all';
+    expIds?: Array<string>;
+    type?: string;
+    releaseChannel?: string;
+    bundleIdentifier?: string;
+    publicUrl?: string;
   } = {}
 ) {
```

Comment:
add return types, especially for exported functions

Reply:
good call